DDDEastAnglia.Tests/SponsorBuilder.cs

```diff
@@ -0,0 +1,54 @@
+using System;
+using DDDEastAnglia.DataAccess.SimpleData.Models;
+
+namespace DDDEastAnglia.Tests
+{
+    internal class SponsorBuilder
+    {
+        private byte[] logo;
+        private string name;
+        private DateTime? paymentDate;
+        private bool showPublicly = true;
+        private int sponsorId;
+        private int sponsorshipAmount;
+        private string url;
+
+        public Sponsor Build()
```

Comment:
> I like the idea of the AtPlatinumLevel() idea.. very nice.

I like the idea too, but wouldn't we need to change the tests when we change the sponsorship amounts? Also, the tests (and the code) don't really care about what level someone has sponsored us at, just how much they have sponsored.